schemas/tls.yml

```diff
@@ -0,0 +1,189 @@
+---
+- name: tls
+  title: TLS
+  group: 2
+  short: Fields describing a TLS connection.
+  description: >
+    Fields related to TLS activity.
+  type: group
+  fields:
+
+    - name: version
+      level: extended
+      type: group
+      description: Fields related to the SSL/TLS version used.
+      fields:
+        - name: original
+          level: extended
+          type: keyword
+          description: Original, unparsed version string provided by generating application.
+
+        - name: protocol
+          level: extended
+          type: keyword
+          description: Normalized protocol name parsed from original string (e.g. ssl, tls).
+
+        - name: number
+          level: extended
+          type: number
```

Comment:
This should probably be treated as a keyword. Otherwise it would need to be indexed as a float.

Reply:
Thanks. Fixed. I just went through this all blindly, and @webmat told me how to run the other tools this morning. 👍